Makefile.tools.mk

```diff
@@ -1,16 +1,192 @@
-LOCALBIN ?= $(shell pwd)/bin
+## Local directories are defined in main Makefile
 $(LOCALBIN):
 	[ -d $@ ] || mkdir -p $@
 
+$(LOCALLIB):
+	[ -d $@ ] || mkdir -p $@
+
 ## Tool binary names.
+GINKGO = $(LOCALBIN)/ginkgo
+GOLANGCI_LINT = $(LOCALBIN)/golangci-lint
+KUSTOMIZE = $(LOCALBIN)/kustomize
 TYPOS = $(LOCALBIN)/typos
+## Dependencies
+TOKENIZER_LIB = $(LOCALLIB)/libtokenizers.a
 
-## Tool versions.
+## Tool fixed versions.
+GINKGO_VERSION ?= v2.27.2
+GOLANGCI_LINT_VERSION ?= v2.1.6
+KUSTOMIZE_VERSION ?= v5.5.0
 TYPOS_VERSION ?= v1.34.0
+VLLM_SIMULATOR_TAG ?= v0.6.1
```

Comment:
I want all of the images setup in the main Makefile

You need to export VLLM_SIMULATOR_IMAGE and no VLLM_SIMULATOR_TAG